code/src/components/Footer.css

```diff
@@ -0,0 +1,80 @@
+.Footer {
+    width: 100%;
+    max-height: 500px;
+    padding: 50px;
+    background-color: #4C5A66;
+    color: var(--main-light-color);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 30px;
+}
+
+.Footer hr {
+    color: var(--main-light-color);
+    border: 0.25px solid var(--main-light-color);
+    opacity: 0.5;
+    width: 80%;
+}
+
+.linkfield {
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+}
+
+.linkfield p {
+    cursor: pointer;
+}
+
+
+.iconfield {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 20px;
+}
+
+.iconfield img {
+    scale: 0.9;
```

Comment:
Instead of scale, use the transform property with the scale() function for better browser support